quickstarts/lambdatest/config.yml

```diff
@@ -0,0 +1,61 @@
+id: f5d5d638-24ee-4a1c-859f-9cbd4d8cabbc
```

Comment:
Please remove this line, new quickstarts will get an ID when they are merged into the `main` branch for the first time.
```suggestion
```

Reply:
Removed